Configuration/FlexForms/flexforms_pi1.xml

```diff
@@ -370,6 +370,7 @@
                             <config>
                                 <type>select</type>
                                 <foreign_table>tx_seminars_event_types</foreign_table>
+                                <foreign_table_where>AND tx_seminars_event_types.pid LIKE ###PAGE_TSCONFIG_STR### ORDER BY title</foreign_table_where>
```

Comment:
Won't this fail if `PAGE_TSCONFIG_STR` is not set (or empty)?

Reply:
No, I've filled the `Configuration/TSconfig/ContentElementWizard.txt` with wildcard (%). So it can't be empty...

Reply:
Ah, okay. Thanks for the explanation!

Reply:
Ok.